diary.c

```diff
@@ -307,52 +339,113 @@ void usage() {
   printf("Edit journal entries from the command line\n");
   printf("\n");
   printf("Options:\n");
-  printf("  -v, --version       : Print diary version\n");
-  printf("  -h, --help          : Show diary help text\n");
-  printf("  -d, --dir DIARY_DIR : Diary storage directory (DIARY_DIR)\n");
+  printf("  -v, --version                 : Print diary version\n");
+  printf("  -h, --help                    : Show diary help text\n");
+  printf("  -d, --dir           DIARY_DIR : Diary storage directory DIARY_DIR\n");
+  printf("  -e, --editor        EDITOR    : Editor to open journal files with\n");
+  printf("  -f, --fmt           FMT       : Date and file format, change with care\n");
+  printf("  -r, --range         RANGE     : RANGE is the number of years to show before/after today's date\n");
+  printf("  -w, --weekday       DAY       : First day of the week, 0 = Sun, 1 = Mon, ..., 6 = Sat\n");
   printf("\n");
   printf("Full docs and keyboard shortcuts: DIARY(1)\n");
```

Comment:
```suggestion
  printf("Full docs and keyboard shortcuts: man \'DIARY(1)\'\n");
```

I think this is friendlier to people new to Linux

Reply:
You are right, thanks. I just pushed a similar commit.